test/data/complete-form-multipleapplicants.json

```diff
@@ -139,7 +137,7 @@
     "total": "216.50",
     "userId": "999999999",
     "paymentId": "1",
-    "paymentReference": "CODE4$$$diedlastname7297$$$CODE5$$$CODE1$CODE2/3",
+    "reference": "CODE4$$$diedlastname7297$$$CODE5$$$CODE1$CODE2/3",
```

Comment:
The style of reference in this test data was originally used to create the payment - the reference as we use it now is to recieve the payment reference after it has been initiated in the Payments system. A more representitive value would be something like "RC-1234-5678-9012-3456".